src/realm/table.cpp

```diff
@@ -2536,8 +2536,68 @@ void Table::update_from_parent(size_t old_baseline) noexcept
     m_alloc.bump_storage_version();
 }
 
+void Table::schema_to_json(std::ostream& out, const std::map<std::string, std::string>& renames) const
+{
+    out << "{";
+    auto name = get_name();
+    if (renames.count(name))
+        name = renames.at(name);
+    out << "\"name\":\"" << name << "\"";
```

Comment:
I would opt for this to be `out << "\"title\":\"" << name << "\"";` because we use title to describe the name of a table, but if this is more idiomatic for realm then this also LGTM

Reply:
I think we have used "name" historically.